README.md

```diff
@@ -115,6 +115,18 @@ mvn clean install
 
 After building, the plugin jar can be found in `sonar-delphi-plugin/target`.
 
+### Integration Tests
```

Comment:
I think this might be a little too prominent, particularly with the callout. A single sentence like "If you are using JDK 17, integration tests can be run by passing `-DskipTests=false` as a parameter." or similar might get the same meaning across in a more concise way.

The other option is to not say anything - I'm on the fence about whether it's important enough to be in the README at all.